src/pages/index.tsx

```diff
@@ -1,33 +1,47 @@
 import Image from "next/image";
 import { Inter } from "next/font/google";
-import { useEffect, useState } from "react";
+import React, { useEffect, useState } from "react";
 import { Transition } from "@headlessui/react";
 import {
   BsChevronCompactLeft,
   BsChevronCompactRight,
   BsChevronCompactDown,
 } from "react-icons/bs";
 import { RxDotFilled } from "react-icons/rx";
+import ProductCard from "./ProductCard";
+import Header from "/.header";
+// import {
+//   slidesSetOne,
+//   slidesSetTwo,
+//   headerSubSections,
+//   headerSubSectionsContentWallets,
+//   headerSubSectionsContentBags,
+//   whyDoYouCarryContent,
+//   cardsOnlyProduct,
+//   cardsAndBillsProduct,
+//   cardsBillsAndCoinsProduct,
+// } from "./data";
 
 export default function Home() {
   const [sliderValue, setSliderValue] = useState("0");
   const [isOpen, setIsOpen] = useState(false);
   const [slidePicture, setsliderPicture] = useState(false);
   const [imagesToShow, setImagesToShow] = useState("Cards only");
+  const [walletDataSet, setWalletDataSet] = useState(cardsOnlyProduct);
 
   const handleSlide = () => {
     setsliderPicture(!slidePicture);
   };
 
-  const changeCard = (imagesToShow) => {
+  useEffect(() => {
     if (imagesToShow === "Cards only") {
-      return cardsOnlyProduct;
+      setWalletDataSet(cardsOnlyProduct);
     } else if (imagesToShow === "Cards and bills") {
-      return cardsAndBillsProduct;
+      setWalletDataSet(cardsAndBillsProduct);
     } else if (imagesToShow === "Cards, bills and coins") {
```

Comment:
I would update the state values, using space is not recommended

Reply:
not sure how to remove the spaces without refactoring the imagesToShow function to check for a different type of string without spaces. This would lead me to add key pair inside of the obj. If that was the case, would it be better to add an ID instead of string to check?

Reply:
please clarify if this approach is correct, or another course of action is better.